docs/_interface/mining.md

```diff
@@ -47,6 +47,29 @@ Now start geth and wait for it to sync the blockchain. This will take quite a wh
 
     geth --http --miner.etherbase 0xC95767AC46EA2A9162F0734651d6cF17e5BfcF10
 
+To monitor the syncing, in another terminal you can `attach` the geth JavaScript console to the running node like so:
+
+    geth attach https://127.0.0.1:8545
+
+and then at the > prompt type
+
+    eth.syncing
+
+You'll see something like the example output below -- it's a two stage process as described in much more detail in our [FAQ](../FAQ).
+In the first stage, the difference between the "currentBlock" and the "highestBlock" will decrease until they are almost equal. It will then look stuck and appear as never becoming equal.  But you should see "pulledStates" rising to equal "knownStates." When both are equal, you are synced.
+
+Example output of first stage of block downloading:
+
+    {
+      currentBlock: 10707814,
+      highestBlock: 13252182,
+      knownStates: 0,
+      pulledStates: 0,
+      startingBlock: 3809258
+}
+
+A response from `eth.syncing` of `false` means you are synced up.
```

Comment:
a `false` could also mean that you are not syncing and not synced up?

Reply:
ah, very good point.  You can fix that perhaps by expanding that sentence I added.
But that sentence is not the API reference itself which covers that detail and doesn't say much:  https://eth.wiki/json-rpc/API#eth_syncing  <-- could probably be improved by someone